faker/providers/address/en_US/__init__.py

```diff
@@ -279,6 +279,26 @@ class Provider(AddressProvider):
         'WV', 'WI', 'WY',
     )
 
+    states_postcode = {
+        'AL': (35004,36925), 'AK': (99501, 99950), 'AZ': (85001,86556),
```

Comment:
PEP8: can you add a space after every comma in the tuples? eg: `'AL': (35004, 36925)`